libs/platform/i18n/src/lib/locale/default-locale.service.ts

```diff
@@ -18,6 +18,9 @@ export class DefaultLocaleService implements LocaleService {
   protected setActive$ = new ReplaySubject<string>(1);
   protected active$;
   protected all$;
+  private dateFormat$: Observable<Intl.DateTimeFormat>;
+  private timeFormat$: Observable<Intl.DateTimeFormat>;
+  private dateTimeFormat$: Observable<Intl.DateTimeFormat>;
```

Comment:
We always use `protected` access modifier for extendibility purpose:
```suggestion
  protected dateFormat$: Observable<Intl.DateTimeFormat>;
  protected timeFormat$: Observable<Intl.DateTimeFormat>;
  protected dateTimeFormat$: Observable<Intl.DateTimeFormat>;
```